nilearn/surface/surface.py

```diff
@@ -1492,6 +1492,13 @@ def to_filename(self, filename):
 
         _data_to_gifti(data, filename)
 
+    def _dtype(self):
+        """Return dtype of the first part.
+
+        Assume all parts have same dtype.
```

Comment:
We may want to ensure that SurfaceImages have similar type for both hemispheres, no?

Reply:
done in #5530